phalcon/application.zep

```diff
@@ -118,13 +118,31 @@ abstract class Application extends Injectable implements EventsAwareInterface
 	public function getModule(string! name) -> array | object
 	{
 		var module;
+		var exception;
 
 		if !fetch module, this->_modules[name] {
-			throw new Exception("Module '" . name . "' isn't registered in the application container");
+			let exception = new Exception("Module '" . name . "' isn't registered in the application container");
+  			this->_handleException(exception);
```

Comment:
Yes im sure, there is just missing return keyword. if exception will be handled like https://github.com/phalcon/cphalcon/pull/12746/files#diff-9721271d9a975486eef19f86a36c1fe9R139 here returned false the execution of code will just go on and it will return not set module value.